src/telepathy/telepathy-stubs.c

```diff
@@ -303,19 +303,6 @@ void sipe_backend_notify_error(SIPE_UNUSED_PARAMETER struct sipe_core_public *si
 			       SIPE_UNUSED_PARAMETER const gchar *title,
 			       SIPE_UNUSED_PARAMETER const gchar *msg) {}
 
-/** USER *********************************************************************/
-
-void sipe_backend_user_feedback_typing(SIPE_UNUSED_PARAMETER struct sipe_core_public *sipe_public,
```

Comment:
Where exactly do you see these implemented? I did a grep of the source code and this is the only place where I see the functions mentioned in `telepathy/` subdir.